### PR TITLE
Add stability test for multiple drainers and reparo restore

### LINKED ISSUE #748: Stability test and e2e test for multiple drainers
## Feature Request
follow up #744

### PR DESCRIPTION
**Pull request by aylei**
Signed-off-by: Aylei <rayingecho@gmail.com>

close #748 

### Check List 

Tests 

 - Stability test

Code changes

 - Has Helm charts change
 - Has Go code change
 - Has CI related scripts change
 - Has documents change

Side effects

 - Breaking backward compatibility

Related changes

 - Need to cherry-pick to the release branch
 - Need to update the documentation

### Does this PR introduce a user-facing change?:
 No.

### CODE REVIEW

**Comment by aylei** [comment]
@tennix @weekface @xiaojingchen @shuijing198799 PTAL

File: tests/backup.go [hunk 1]
```diff
@@ -1,73 +1,221 @@
...
-	err = oa.CheckIncrementalBackup(from, false)
+	err = oa.DeployAdHocBackup(source)
 	if err != nil {
+		glog.Errorf("cluster:[%s] deploy happen error: %v", source.ClusterName, err)
 		return err
 	}
 
-	err = oa.DeployAdHocBackup(from)
+	// Restoring via reparo is slow, so we stop inserting data as early as possible to reduce the size of incremental data
+	oa.StopInsertDataTo(source)
```
Comment (weekface) [inline tests/backup.go]: This is not the real use case, the backup must run without stopping insert.

**Comment by aylei** [comment]
@weekface Addressed PTAL

**Pull request merged**